engine/src/page.js

```diff
@@ -333,22 +330,24 @@ class Page {
       const stateResult = this.validateState(newState)
 
       if (stateResult.errors) {
-        const viewModel = this.getViewModel(payload, stateResult.errors)
+        const viewModel = this.getViewModel(payload, num, stateResult.errors)
         viewModel.backLink = progress[progress.length - 2]
         return h.view(this.viewName, viewModel)
       }
 
+
       let update = this.getPartialMergeState(stateResult.value)
       if (this.repeatField) {
-        const updateValue = { [this.path]: update[this.section.name] }
-        const sectionState = state[this.section.name]
+        let updateValue = {[this.path]: update[this.section.name]}
+        let sectionState = state[this.section.name]
         if (!sectionState) {
           update = { [this.section.name]: [updateValue] }
-        } else if (!sectionState[num - 1]) {
+        } else if(!sectionState[num-1]) {
```

Comment:
Auto formatting again? Spacing heere is good - don't we have a standard formatter?

Reply:
maybe merge issue, sorry! Might be that engine doesn't have linter installed and/or not running on test. I'll add.